bin/sensei.js

```diff
@@ -12,27 +12,23 @@ const yargs = require("yargs/yargs");
 async function cli(args, env) {
   const {
     _: [command],
-    ...cliOptions
+    $0: _,
+    ...options
   } = args;
-  const material = path.resolve(cliOptions.material);
-  const slug = cliOptions.slug || path.basename(material);
-  const buildOptions = {
-    ...cliOptions,
-    material,
-    slug,
-  };
 
-  console.info(`Processing folder '${material}' as '${slug}'`);
+  console.info(
+    `Processing folder '${options.material}' as '${options.slug}' in format ${options.slideWidth}x${options.slideHeight}`
```

Comment:
Not sure adding the slide size if relevant here, as most of the time it will be the default value.

Reply:
It doesn't cost much and can help now if there is an issue in the config and/or options;

Reply:
Maybe "in format" should be "using slide size" then?

Reply:
Changed.